adb_graphics/figures/maps.py

```diff
@@ -281,60 +282,11 @@ def draw(self, show=False): # pylint: disable=too-many-locals, too-many-branches
 
         # Contour secondary fields, if requested
         if self.contour_fields:
-            for contour_field in self.contour_fields:
```

Comment:
All this is related to linting and having too many statements in the "draw" method.